### ISSUE TITLE
PostPersistEvent - response->setSuccess(false) - undefined method

### ISSUE THREAD

**Issue by lsv** [1up-lab/OneupUploaderBundle#207]
In my listener

```yml
postuploadlistener:
    class: AppBundle\Listener\FilePostUploadListener
    tags:
      - { name: kernel.event_listener, event: oneup_uploader.post_persist, method: onUpload }
```

```php
public function onUpload(PostPersistEvent $event)
{
    $response = $event->getResponse();
    $response->setSuccess(false);
    $response->setError('RandomError');
}
```

Im getting this

```
Attempted to call an undefined method named "setSuccess" of class "Oneup\UploaderBundle\Uploader\Response\EmptyResponse
```

So setSuccess does not exists in the EmptyResponse class.
So it seems that the [documentation](https://github.com/1up-lab/OneupUploaderBundle/blob/master/Resources/doc/response.md) has some error in it?

Version: v1.3.1
Hash: 00bff0181fc1592134f1aa36469ea44d69585a9a

Using dropzone

**Comment by lsv** [comment]
Also tried to make a validator

```yml
postuploadlistener:
    class: AppBundle\Listener\FilePostUploadListener
    tags:
      - { name: kernel.event_listener, event: oneup_uploader.validation, method: onValidate }
```

```php
public function onValidate(ValidationEvent $event)
{
    throw new ValidationException('Denied');
}
```

But it looks like the validator gets ignored, the file gets uploaded and frontend shows OK on the file

**Comment by lsv** [comment]
By updating to dev-release-1.4.0 it now works.

My validation method looks like this

```php
public function onValidate(ValidationEvent $event)
    {
        if ($event->getFile()->getExtension() != 'pdf') {
            throw new ValidationException('Not a PDF file');
        }
    }
```

And my onUpload method

```php
public function onUpload(PostPersistEvent $event)
    {
        throw new ValidationException('Whoops');
    }
```

Maybe update the documentation also?

**Comment by bytehead** [comment]
I've just released [1.4.0](https://github.com/1up-lab/OneupUploaderBundle/releases/tag/1.4.0).
Thanks again for your contribution in #202, #208 and #209.
